src/main/etc/default/ike.prod

```diff
@@ -0,0 +1,3 @@
+START=yes
```

Comment:
Delete START (unnecessary?) and APPROOT (redundant, though it can be overriden here)

Also do the same for the template